net-misc/econnman/econnman-1.1.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Connman user interface written with EFL & python"
+HOMEPAGE="https://phab.enlightenment.org/w/projects/econnman/ https://www.enlightenment.org/"
+SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${P}.tar.xz"
+
+LICENSE="LGPL-3"
```

Comment:
You sure it's not 3+? I don't really want to check ;-P.

Reply:
But you're going to have to, since I can't tell the difference between those two ;) 
I presume LGPL-3+ is LGPL-3 with words like "version x **or later**" / "version x **-plus**", anyway here's the LICENSE file:
https://git.enlightenment.org/apps/econnman.git/tree/COPYING

It's identical to `/usr/portage/licenses/LGPL-3`, and to my best knowledge, LGPL-3.

Gonna fix the python-thing, it does check for main interpreter in configure (my bad).

Reply:
Look at source files (`.c`), there should be a copyright notice on top.

Reply:
There are no source files, this is basically a runnable python-script + theme file + docs, that's why I probably missed the python req in the first time. The theme file is compiled with *edje_cc* during `src_compile`, but nothing else happens there.

Reply:
Ok, if there's no file with copyright notice, then I guess that's the best we can state.